lsp-pyright.el

```diff
@@ -159,35 +183,49 @@ Current LSP WORKSPACE should be passed in."
 (lsp-register-custom-settings
  `(("pyright.disableLanguageServices" lsp-pyright-disable-language-services t)
    ("pyright.disableOrganizeImports" lsp-pyright-disable-organize-imports t)
+   ("python.analysis.autoImportCompletions" lsp-pyright-auto-import-completions t)
+   ("python.analysis.typeshedPaths" lsp-pyright-typeshed-paths)
+   ("python.analysis.stubPath" lsp-pyright-stub-path)
    ("python.analysis.useLibraryCodeForTypes" lsp-pyright-use-library-code-for-types t)
    ("python.analysis.diagnosticMode" lsp-pyright-diagnostic-mode)
    ("python.analysis.typeCheckingMode" lsp-pyright-typechecking-mode)
    ("python.analysis.logLevel" lsp-pyright-log-level)
    ("python.analysis.autoSearchPaths" lsp-pyright-auto-search-paths)
    ("python.analysis.extraPaths" lsp-pyright-extra-paths)
-   ("python.pythonPath" lsp-pyright-locate-python)))
+   ("python.pythonPath" lsp-pyright-locate-python)
+   ("python.venvPath" lsp-pyright-venv-path)))
```

Comment:
I still suggest to use `lsp-pyright-locate-venv` to get the `venv-path` automatically as `lsp-pyright-locate-python`.

Reply:
And also make `lsp-pyright-venv-path` as buffer-local variable.

Reply:
Thanks added `locate-venv`.

`venvPath` is set only during `initialize`, do we need to make it buffer local?

Reply:
It's better be, but the server has to be restarted. But i's fine with me for now.